src/swap/types.ts

```diff
@@ -3,47 +3,82 @@ import { Token } from '../token/types';
 
 export type AddressOrETH = Address | 'ETH';
 
+/**
+ * Note: exported as public Type
+ */
 export type Fee = {
-  amount: string;
-  baseAsset: Token;
-  percentage: string;
+  amount: string; // The amount of the fee
+  baseAsset: Token; // The base asset for the fee
+  percentage: string; // The percentage of the fee
 };
 
+export type GetQuoteAPIParams = {
+  from: AddressOrETH | ''; // The source address or 'ETH' for Ethereum
+  to: AddressOrETH | ''; // The destination address or 'ETH' for Ethereum
+  amount: string; // The amount to be swapped
+  amountReference?: string; // The reference amount for the swap
+};
+
+/**
+ * Note: exported as public Type
+ */
+export type GetQuoteParams = {
+  from: Token; // The source token for the swap
+  to: Token; // The destination token for the swap
+  amount: string; // The amount to be swapped
+  amountReference?: string; // The reference amount for the swap
```

Comment:
What's a `reference`?

Reply:
whether the `amount` in the parameter is referring to the `from` token or the `to` token (e.g. whehter you are trying to swap to 1 ETH or 1 DEGEN)

Reply:
ok in the next PR let's make this more clear in the comments and docs.

Folks should find easy to learn how this works just by reading the types.